package.json

```diff
@@ -70,6 +70,8 @@
     "@types/react": "17.0.47",
     "**/@angular-devkit/build-angular/minimatch": "3.0.5",
     "**/@angular-devkit/build-angular/webpack": "^5.76.0",
+    "**/@changesets/cli/**/normalize-package-data/semver": "7.5.4",
```

Comment:
Think we can remove this too actually: https://github.com/npm/normalize-package-data/issues/182, and it might be just be that our lockfile is outdated.

What happens if you remove this and run `yarn install` again?

Reply:
good call! forgot that we could use 5.7.2 so that seemed to work :)